whelk-core/src/main/groovy/whelk/component/ElasticSearch.groovy

```diff
@@ -24,7 +24,7 @@ class ElasticSearch {
     static final String BULK_CONTENT_TYPE = "application/x-ndjson"
     static final String SEARCH_TYPE = "dfs_query_then_fetch"
 
-    static final Set<String> LANGUAGES_TO_INDEX = ['sv', 'en'] as Set
+    // FIXME: de-KBV/Libris-ify: configurable
     static final List<String> REMOVABLE_BASE_URIS = [
             'http://libris.kb.se/',
             'https://libris.kb.se/',
```

Comment:
We are just imitating the cataloging interface's way of generating a fallback label here.
Maybe it can be removed?
(or consolidated with site_config)